vendor/bower/icono/sass/generals.scss

```diff
@@ -0,0 +1,42 @@
+.icono-stroke{
+	border: $U2 solid;
+}
+
+.stickCenterV{
+    @include stickCenterV;
+}
+
+.stickCenterH{
+    @include stickCenterH;
+}
+
+.stickCenter{
+    @include stickCenter;
+}
+
+.spin[class*="spin"]{
+    -webkit-animation: loading-spinner 2s infinite linear; /* Safari 4+ */
```

Comment:
Use `//` comments everywhere<br>Line should be indented 2 spaces, but was indented 4 spaces<br>Properties should be ordered -moz-animation, -o-animation, -webkit-animation, animation<br>Avoid vendor prefixes.